src/main.rs

```diff
@@ -140,6 +210,26 @@ async fn main() -> ExitCode {
         config
     );
 
+    if let Some(Command::Clean { provision, logs }) = &opts.command {
```

Comment:
nitpick, but I _believe_ you can drop the reference here and use `opts.command` so you later don't need to dereference the bools in your if statements.